.github/workflows/build-ui-and-server.yml

```diff
@@ -177,10 +177,11 @@ jobs:
           PORT: 9081
           ADAPTER_URLS: "localhost:10000 localhost:10001 localhost:10002 localhost:10003 localhost:10004 localhost:10009 localhost:10007"
         run: |
-          make server &
-          sleep 60
+          make server &&
+          sleep 90
```

Comment:
While we should probably avoid using `sleep` and instead be event-driven, I understand that this can be a challenge to do.